test_fixtures/generation/copynet/experiment_transformer.jsonnet

```diff
@@ -0,0 +1,69 @@
+local target_namespace = "target_tokens";
+local transformer_model = "bert-base-cased";
```

Comment:
There are smaller ones that you can use here, to make this faster.  E.g., see [here](https://github.com/allenai/allennlp-guide/blob/f2eb4c91f6576ecea639171d3a98f9e0846702a0/exercises/part2/representing-text-as-features/token_embedders_contextual.py#L29-L31).

Reply:
I'm getting some weird errors when trying to use that model:

<img width="1479" alt="image" src="https://user-images.githubusercontent.com/8812459/85083780-9b27ce80-b187-11ea-854c-4dbf57fdae06.png">